weasyl/ads.py

```diff
@@ -0,0 +1,112 @@
+import os
+import random
+import time
+
+from datetime import datetime
```

Comment:
PEP 8, Imports, https://www.python.org/dev/peps/pep-0008/#imports : 

> "Imports should be grouped in the following order: [...] 1) standard library imports [...]"

Shouldn't this be in the above block with ``os``, ``random``, and ``time``?